Automattic/wp-calypso
# Add Offer Jetpack Complete page main component & layout.

#### Proposed Changes

This PR creates the a JetpackRnaDialogCard component that can be reused for similar RNA cards with an RNA image on the left and product content on the right.  The PR just creates the basic component and layout, whereas the content body will still need to be added in.
See Figma designs for final designs: R1U6Ww74mdgMrKUJ21BnTL-fi-3061%3A11111

Completes task: 1203759331596262-as-1203791007934498/f

**Desktop view:**

![Screenshot on 2023-01-27 at 19-56-59](https://user-images.githubusercontent.com/11078128/215232935-7349bb54-4212-48a0-94f7-50a5c81cd5a0.png)

.
**Mobile View:**

![Screenshot on 2023-01-27 at 19-58-32](https://user-images.githubusercontent.com/11078128/215232964-20c0f059-7a86-4a68-ac73-bed8ad00f7ca.png)

#### Testing Instructions

* boot up this PR 
    * Run `git fetch && git checkout add/jp-complete-page-layout-component`
    * Run `yarn start`
    * Goto http://calypso.localhost:3000
* OR use the Live Link provided below
* Create a new JN site
* Navigate to `/jetpack/connect/plans/complete/[JN Site]`.  If using the Live Link, please append `?flags=jetpack/offer-complete-after-activation` to the url.
* Verify you can see the main Jetpack Complete component as shown in the screenshots below.
* If you have an unactivated product license key available, you should see the "Already have an existing plan or license key? Click here to get started" (desktop view). Or "Activate license" (mobile view).  Test that this message shows if a licence key is available and that it does not show if no license keys are available.
* View in mobile and desktop views. Compare with figma designs:  R1U6Ww74mdgMrKUJ21BnTL-fi-3061%3A11111



*

#### Pre-merge Checklist





Related to #

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - 0-as-1203791007934498
